gui/src/components/StoryComponents/StoryDetails.tsx

```diff
@@ -43,6 +43,11 @@ export default function StoryDetails({
     setEditTrue,
   } = useBoardContext();
   const router = useRouter();
+
+  const [issueTitle, setIssueTitle] = useState<string>('');
+  const [issueDescription, setIssueDescription] = useState<string>('');
+  const [actions, setActions] = useState<{ label: string; link: string }[]>([]);
```

Comment:
enable prettier and run the lint command on the terminal before pushing the code